src/Xamarin.Android.Build.Tasks/Utilities/TypeMapGenerator.cs

```diff
@@ -131,8 +131,10 @@ public ReleaseGenerationState ()
 
 		public TypeMapGenerator (TaskLoggingHelper log, ITypeMapGeneratorAdapter state, AndroidRuntime runtime)
 		{
-			this.log = log ?? throw new ArgumentNullException (nameof (log));
-			this.state = state ?? throw new ArgumentNullException (nameof (state));
+			ArgumentNullException.ThrowIfNull (log);
+			ArgumentNullException.ThrowIfNull (state);
```

Comment:
Undo this change `ArgumentNullException.ThrowIfNull()` does not exist in `netstandard2.0`.